modules/logging-release-notes-5-8-15.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-15_{context}"]
+= Logging 5.8.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:140310[RHBA-2024:140310].
```

Comment:
These links are still 404ing. Has it not actually been released yet, or is something wrong with the URLs?

Reply:
Something went wrong with the actual release, several times. Still pending.

Reply:
This one is still 404ing: https://access.redhat.com/errata/RHBA-2024:140310

I'll put it back into the queue until it's ready to go

Reply:
@libander Should this be:
```suggestion
This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:10053[RHBA-2024:10053].
```
Looks like it got the new advisory number of `10053` when it shipped, and that one is available/live: https://access.redhat.com/errata/RHBA-2024:10053

Reply:
Sorry, I missed the second advisory link during the original review and only checked the first one. 10052 and 10053 are the correct numbers.